src/styles/main.scss

```diff
@@ -1 +1,76 @@
 @use "@nationalarchives/frontend/nationalarchives/all";
+
+#search-form {
+  padding-bottom: 2em;
+}
```

Comment:
Instead of this CSS, could you please add a class of `tna-!--margin-bottom-m` to the `#search-form` element?